### ISSUE TITLE
Export Settings

### ISSUE THREAD

**Issue by GS2020-2020** [pawelsalawa/sqlitestudio#5106]
### Details
Here you describe your problem details, or proposal for enhancement.

Ability to export settings

**Comment by marcpley** [comment]
So import settings too ?
When designing a DB using functions, it seems that those functions are not stored in the DB but in the settings. So a settings file must be added to the DB file for new users of the DB.
Is there a way to store the settings, or only the functions, in the DB ?

**Comment by pawelsalawa** [comment]
@marcpley Do you ask whether Custom SQL Functions are stored in settings or in the DB? The answer is - they are stored in settings and it cannot be the other way around. SQLite does not support stored functions. There has to be application, which registers additional functions in the open, ongoing connection to the database. It's completely up to the application that uses SQLite database how these functions are executed. SQLiteStudio is executing them. They cannot be simple "transfered" to the target database and moved with database file, because execution engine of custom functions is in the application, not in the SQLite library.

**Comment by marcpley** [comment]
Ok, thanks for the clarification :-)
So we have to give a settings file with the DB file. If we can choose the name of the settings file it can have the same name of the DB file with a different ext.
And if we can import some settings but not all, it will be great, for example the functions and not the color theme :-)

**Comment by pawelsalawa** [comment]
Yes, I believe this is what this issue is about - to be able to export some settings (like functions) and then import them elsewhere.
